extensions/amp-accordion/0.1/amp-accordion.css

```diff
@@ -30,16 +30,13 @@
   border: solid 1px #dfdfdf;
 }
 
-amp-accordion > section:not([expanded]) > :last-child[rendersubtree] {
-  border: none !important;
+amp-accordion.i-amphtml-display-locking > section:not([expanded]) > :last-child,
+amp-accordion.i-amphtml-display-locking
+  > section:not([expanded])
+  > :last-child
+  * {
+  content-visibility: hidden-matchable !important;
```

Comment:
And this solves all the other things we had with `position:absolute`?

Reply:
If you're referring to the prior-needed hack for IntersectionObserver and the analytics problem, I was going to hold off on testing until the scrolling issue is resolved.